hrjobcontract/js/test/controllers/modal/modal-contract-new.controller.spec.js

```diff
@@ -86,6 +89,61 @@ define([
       });
     });
 
+    describe('when user clicks on the wrench icon', function () {
+      locationUrl = '/civicrm/hours_location?reset=1';
```

Comment:
Should we split this into multiple suites? One test is just too large and it is a bit difficult to follow what's going on here.

There should be multiple describes, smth like `when user clicks on the "hours location" wrench icon`. etc